Update tutorial-deploy-machine-learning.md

dariuszparys opened the pull request:
Either change text to point to `amlInput` or change route definition to `mlInput`

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

dariuszparys committed:
Update tutorial-deploy-machine-learning.md

Either change text to point to `amlInput` or change route definition to `mlInput`
